### PR TITLE
Added .wslconfig changes to wsl-config.md

### PR DESCRIPTION
**Pull request by craigloewen-msft**
Added info about .wslconfig to wsl-config.md

### CODE REVIEW

File: WSL/wsl-config.md [hunk 1]
```diff
@@ -291,3 +291,37 @@ These options are available in Insider Build 17713 and later.
...
+You can configure global WSL options by placing a `.wslconfig` file into the root directory of your users folder: `C:\Users\<yourUserName>\.wslconfig`. This file can contain the following options:
+
+### WSL 2 Settings
+
+These settings affect the VM that powers any WSL 2 distribution. 
+
+| key | value | default | notes|
+|:----|:----|:----|:----|
+| kernel | string | The Microsoft built kernel provided inbox | An absolute Windows path to a custom Linux kernel. |
+| memory | size | 70% of your total memory on Windows | How much memory to assign to the WSL 2 VM. |
```
Comment (craigloewen-msft) [inline WSL/wsl-config.md:318]: @benhillis can you double check the default value here?

File: WSL/wsl-config.md [hunk 2]
```diff
@@ -291,3 +291,37 @@ These options are available in Insider Build 17713 and later.
...
+| processors | number | The same number of processors on Windows | How many processors to assign ot the WSL 2 VM. |
+| localhostForwarding | boolean | `true` | Boolean specifying if ports bound to wildcard or localhost in the WSL 2 VM should be connectable from the host via localhost:port. |
+| kernelCommandLine | string | Blank | Additional kernel command line arguments. |
+| swap | size | ??? 30GB? | How much swap space to add to the WSL 2 VM, 0 for no swap file. |
```
Comment (craigloewen-msft) [inline WSL/wsl-config.md:322]: @benhillis Can you check the default here too?

File: WSL/wsl-config.md [hunk 3]
```diff
@@ -291,3 +291,37 @@ These options are available in Insider Build 17713 and later.
...
+| swapFile | size | Same directory as your distro??? | An absolute Windows path to the swap virtual hard disk. |
```
Comment (craigloewen-msft) [inline WSL/wsl-config.md:323]: @benhillis and last one here :)

File: WSL/wsl-config.md (hunk 1 above)
Comment (benhillis) [reply WSL/wsl-config.md:318]: 80%

File: WSL/wsl-config.md (hunk 2 above)
Comment (benhillis) [reply WSL/wsl-config.md:322]: 25% the memory size rounded up to the nearest GB.

File: WSL/wsl-config.md (hunk 3 above)
Comment (benhillis) [reply WSL/wsl-config.md:323]: %USERPROFILE%\AppData\Local\Temp\swap.vhdx

**Pull request merged**